cli/src/program.rs

```diff
@@ -1543,7 +1581,7 @@ fn process_write_buffer(
     let fee_payer_signer = config.signers[fee_payer_signer_index];
     let buffer_authority = config.signers[buffer_authority_signer_index];
 
-    let program_data = read_and_verify_elf(program_location)?;
+    let program_data = read_and_verify_elf(program_location, FeatureSet::all_enabled())?;
```

Comment:
This can be done in a follow-up, but what do you think about also having `write-buffer` do a verification and allow for skipping, just like deploy?

Reply:
Will do.

Reply:
The follow up is #1794.